app/src/pages/inside/stepPage/modals/makeDecisionModal/optionsSection/itemsList/itemsListBody/itemsListBody.jsx

```diff
@@ -114,35 +87,31 @@ SimilarItemsList.propTypes = {
   selectItem: PropTypes.func.isRequired,
   showErrorLogs: PropTypes.bool.isRequired,
   isBulkOperation: PropTypes.bool,
-  isNarrowView: PropTypes.bool,
-  eventsInfo: PropTypes.object,
   onClickExternalLinkEvent: PropTypes.func,
+  eventsInfo: PropTypes.object,
 };
 SimilarItemsList.defaultProps = {
-  isNarrowView: false,
-  eventsInfo: {},
   onClickExternalLinkEvent: () => {},
+  eventsInfo: {},
 };
 
 const HistoryLineItemsList = ({
   testItems,
   selectedItems,
   selectItem,
-  isNarrowView,
   onClickExternalLinkEvent,
 }) => {
   return (
     testItems.length > 0 &&
     testItems.map((item, i) => {
       return (
-        <ExecutionInfo
+        <TestItemDetails
           item={item}
           selectItem={i !== 0 ? selectItem : undefined}
           isSelected={!!selectedItems.find((selectedItem) => selectedItem.id === item.id)}
-          preselected={i === 0}
           key={item.id}
-          isNarrowView={isNarrowView}
           onClickLinkEvent={onClickExternalLinkEvent}
+          mode={HISTORY_LINE_ITEM}
```

Comment:
Please recheck history line item with long test item name it is overflowing. Also clarify state if item has some labels(f.e AA).
Whether the history item should display a comment and error log when clicked?